### PR TITLE
Toggling 1 detection announcing 1 detection and disabling 9 detection in SDD

### PR DESCRIPTION
**Pull request by unknownhad**
Toggling 1 detection announcing 1 detection and disabling 9 detection in SDD

### CODE REVIEW

**Review by vs-mg — APPROVED** [review]

File: content/waf/change-log/scheduled-changes.md [hunk 1]
```diff
@@ -23,14 +23,95 @@ layout: list
...
+      <td>This detection will replace 100005 on legacy WAF and ...1bc977d1 on new WAF</td>
+    </tr>
+    <tr>
+      <td>2022-10-10</td>
+      <td>2022-10-17</td>
+      <td>Disable</td>
+      <td>N/A</td>
+      <td>...eebf3863</td>
+      <td>California Driver's Licence</td>
+      <td>This detection is part of Senstive Data Disclosure (SDD)</td>
```
Comment (pedrosousa) [inline content/waf/change-log/scheduled-changes.md]:
```suggestion
      <td>This detection is part of Sensitive Data Disclosure (SDD)</td>
```
Occurs on many table rows.

File: content/waf/change-log/scheduled-changes.md [hunk 2]
```diff
@@ -23,14 +23,95 @@ layout: list
...
+      <td>US Passport</td>
+      <td>This detection is part of Senstive Data Disclosure (SDD)</td>
+    </tr>
+    <tr>
+      <td>2022-10-10</td>
+      <td>2022-10-17</td>
+      <td>Disable</td>
+      <td>N/A</td>
+      <td>...0a290153</td>
+      <td>Wisonconsin Driver's Licence</td>
```
Comment (pedrosousa) [inline content/waf/change-log/scheduled-changes.md]:
```suggestion
      <td>Wisconsin Driver's Licence</td>
```
Should this be **Wisconsin**?

**Review by vs-mg — APPROVED** [review]

File: content/waf/change-log/scheduled-changes.md (hunk 2 above)
Comment (unknownhad) [reply content/waf/change-log/scheduled-changes.md]: proposed changes fixed along with 1 more change.

File: content/waf/change-log/scheduled-changes.md [hunk 3]
```diff
@@ -23,14 +23,95 @@ layout: list
...
+      <td>Illinois Driver's Licence</td>
+      <td>This detection is part of Sensitive Data Disclosure (SDD)</td>
+    </tr>
+    <tr>
+      <td>2022-10-10</td>
+      <td>2022-10-17</td>
+      <td>Disable</td>
+      <td>N/A</td>
+      <td>...9f7200b4</td>
+      <td>New York Driver's licence</td>
```
Comment (vs-mg) [inline content/waf/change-log/scheduled-changes.md:67]: Licence?

Comment (unknownhad) [reply content/waf/change-log/scheduled-changes.md:67]: US V/S UK :)

Comment (vs-mg) [reply content/waf/change-log/scheduled-changes.md:67]: nit: I meant `New York Driver's Licence` instead of `New York Driver's licence`

**Pull request merged**